ext/Kconfig

```diff
@@ -14,4 +14,6 @@ source "ext/lib/crypto/Kconfig"
 
```

Comment:
I wonder if we really need to have another base64 library as we have base64 support functions already in mbedtls. One is be able to use them without embedding full mbedtls library.

Reply:
Surely i was thinking about such option. What i think is that mbetls is another external library, not targeted to provide this encoding. So base64 from mbedtls might changes or disappears without giving us warning prior. (mbedtls is under arm subsidiary control) Is better to have base64 separately.

Reply:
> So base64 from mbedtls might changes or disappears without giving us warning prior.

Not likely as there are already users of base64() in zephyr, and even if mbedtls would remove base64(), we could then add another library implementing that. Anyway, just my random comment to this while looking various PRs.